cli/cook/subcommands/wait.py

```diff
@@ -0,0 +1,61 @@
+import time
+
+from cook.subcommands.show import query, print_no_data, seconds_to_timedelta
```

Comment:
Should these helper function be moved to a `utils.py`?

Reply:
`query` and `print_no_data` are really specific to show and wait, so I left them as is. I did move `seconds_to_timedelta` and its friends into `util`